File: src/Omnia.ProcessManagement.Models/Workflows/WorkflowApprovalTask.cs
using Omnia.Fx.Models.Users;
using Omnia.ProcessManagement.Models.Processes;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.Workflows
{
    public class WorkflowApprovalTask : WorkflowTask
    {
        public WorkflowApprovalTask()
        {

        }

        public WorkflowApprovalTask(WorkflowTask workflowTask) : base(workflowTask)
        {

        }

        public Process Process { get; set; }
        public bool Responsible { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/workflows/WorkflowApprovalTask.ts
import { Process } from '../processes';
import { WorkflowTask } from './WorkflowTask';
import { User } from '@omnia/fx-models';
import { WorkflowApprovalData } from './WorkflowApprovalData';

export interface WorkflowApprovalTask extends WorkflowTask {
    process: Process,
    responsible: boolean,
    assignedTo: User,
    author: User,
}